kbase-extension/static/kbase/js/widgets/appWidgets2/input/fileInput.js

```diff
@@ -88,13 +88,13 @@ define([
 
         function importControlValue() {
             return Promise.try(() => {
-                return Validation.importString(getControlValue());
+                return Validation.importTextString(getControlValue());
```

Comment:
There are a number of places where an `importString` function gets called. These are specific to the validator - `ints` and `floats` have a different meaning for `importString`. This first one uses `importTextString`, later ones will migrate to `importIntString`, `importFloatString`, and other specific names.